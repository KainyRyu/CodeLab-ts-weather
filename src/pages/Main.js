import axios from 'axios';
import React, { useEffect } from 'react';
import { MOCK } from './data';

export default function Main() {
  // const APIKEY = 'bd5de54bb8eed93db96a311f088b9415';
  // const APICALL = `http://api.openweathermap.org/data/2.5/forecast?id=524901&appid=${APIKEY}`;

  // useEffect(() => {
  //   const data = async () => {
  //     let res;
  //     try {
  //       res = await axios.get(APICALL);
  //     } catch (err) {
  //       console.log(err);
  //     }

  //     return res.data;
  //   };
  //   // data();
  // }, [APICALL]);
  console.log(MOCK);

  return (
    <>
      <h1>This is the main page</h1>
    </>
  );
}
