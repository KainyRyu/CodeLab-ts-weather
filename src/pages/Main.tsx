import Current from '../components/Current';

export default function Main() {
  return (
    <>
      <h1>This is the main page</h1>
      <Current />
    </>
  );
}
