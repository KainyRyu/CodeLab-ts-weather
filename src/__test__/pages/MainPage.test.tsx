import react from 'React'
import {render,screen} from '@testing-library/react'
import Main from 'pages/MainPage'

describe('<MainPage/>',()=>{
  it('renders component correctly',()=>{
    render(<Main/>)
  })
})