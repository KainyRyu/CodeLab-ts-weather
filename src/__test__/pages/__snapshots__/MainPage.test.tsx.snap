// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MainPage/> renders component correctly 1`] = `
<div>
  <div
    class="sc-eCImPb kAZuMp"
  >
    <h1>
      TS Weather
    </h1>
    <div
      class="sc-bdvvtL cgThLl"
    >
      <input
        placeholder="add new location"
        type="text"
        value=""
      />
      <button>
        add
      </button>
    </div>
  </div>
</div>
`;
